dev/Common/IsWindowsVersion.h

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#ifndef __ISWINDOWSVERSION_H
+#define __ISWINDOWSVERSION_H
+
+#include <VersionHelpers.h>
+
+namespace WindowsVersion
+{
+VERSIONHELPERAPI IsWindowsVersionOrGreaterEx(
+    const WORD majorVersion,
+    const WORD minorVersion,
+    const WORD servicePackMajor,
```

Comment:
Other readers - `VERSIONHELPERAPI` is already `static`

Reply:
Yep. Surprised me when first cooking the bits but conveniently F12 in VS quickly answered why `static VERSIONHELPERAPI` caused a compiler error